starter-code/iteration-2/main.ts

```diff
@@ -1,7 +1,24 @@
 // Add the reference to the "TodoInterface"
-
+import {TodoInterface} from "./todoInterface"
 // 1. Create a class Todo that implements the Interface created before.
-
+class Todo implements TodoInterface {
+  TodoList: Array<string> = [];
+  addTask(arg: string): number {
+    this.TodoList.push(arg)
+    console.log("Task added!");
```

Comment:
you could show the task with string interpolation:
`    console.log(`Task "${task}" inserted in the list`);
`